packages/lesswrong/server/apolloServer.ts

```diff
@@ -295,7 +296,7 @@ export function startWebserver() {
     response.setHeader("Content-Type", "text/html; charset=utf-8"); // allows compression
 
     const {bundleHash} = getClientBundle();
-    const clientScript = `<script defer src="/js/bundle.js?hash=${bundleHash}"></script>`
+    const clientScript = `<script src="/js/bundle.js?hash=${bundleHash}"></script>`
```

Comment:
This is only a draft PR, but flagging that my understanding is that the `defer` here is pretty essential for time-to-first-paint performance for clients that don't have a cached bundle download.

Reply:
I'm not sure about `defer` exactly but I think I've got a handle on how the bundle affects time to first paint, and found a way to get round it (by waiting until after the first frame before running the startup script), there is some explanation in [this PR](https://github.com/ForumMagnum/ForumMagnum/pull/9114) (although that one doesn't actually contain the solution). If defer is known to make it worse I would guess it's because it causes execution to start a bit sooner so be more likely to hog CPU that could be used for rendering, in which case that solution would still work, but if there is something else that would be good to know